### ISSUE TITLE
Display settings always reset

### ISSUE THREAD

**Issue by RJswap** [jellyfin/jellyfin#5933]
**Describe the bug**
The display option (like display background image, play music theme, the number of item displayed on each page etc..) reset really frequently.

**System (please complete the following information):**
 - OS: [Synology (linux)]
 - Virtualization: [Docker]
 - Clients: [Browser]
 - Browser: [Firefox]
 - Jellyfin Version: [10.7.2]
 - Installed Plugins: [fanart, theme, merge, tvdb]
 - Reverse Proxy: [nginx]
 - Base URL: [none]

**Expected behavior**
Keep the settings after saving them

**Comment by dmitrylyzo** [comment]
`Display missing episodes within seasons` - server side, cross-client option.
`Backdrops`, `Theme songs`, `Library page size` and other display settings are stored in `localStorage` (client side). Clearing it resets these settings.

**Comment by axeII** [comment]
@dmitrylyzo Yup that makes sense when I open new window of jellyfin backdrops are missing. 

Is there a way to set backdrops default on for all clients?

**Comment by dmitrylyzo** [comment]
@axeII Do you mean a new window on the same PC?

Tested using Firefox 88.0.
- Create new profile.
- Set backdrops on.
- Close browser.
- Open browser.
- Backdrops are still there. _And the other settings I turned on._

This doesn't work with incognito window - it doesn't reuse local storage.

> Is there a way to set backdrops default on for all clients?

This can affect performance. Because of this, they are disabled for Tizen and webOS.

**Comment by axeII** [comment]
@dmitrylyzo I know it's a feature and for me it works as designed 🙂 (as you said only for local storage) 

But I wish there was **a option** for set it "default on" _on the server side_. So if I install new client for example (or incognito tab) backdrops would be set on by default. 

Best case scenario this could be set via environment variable 🙏🏻

**Comment by axeII** [comment]
Hmm maybe I should create different issue for this since it's not a bug but _feature_...

**Comment by crobibero** [comment]
Feature requests belong at https://features.jellyfin.org/, so please don’t create an issue

**Comment by axeII** [comment]
Ah ok @crobibero.

**Comment by Bond-009** [comment]
Feature request
